clinvar_ingest/cloud/bigquery/bq_json_schemas/clinical_assertion.bq.json

```diff
@@ -0,0 +1,109 @@
+[
+  {
+    "name": "datarepo_row_id",
+    "type": "STRING"
```

Comment:
Are we still keeping the datarepo_row_id column? if so, what values are we planning on putting in it. If not, should we just drop it?

Reply:
No, good point. They all come through as `null` since nothing is set for that field. I'll remove this from all the tables